src/vulkan/engine_vulkan.cc

```diff
@@ -51,6 +51,40 @@ VkShaderStageFlagBits ToVkShaderStage(ShaderType type) {
   return VK_SHADER_STAGE_FRAGMENT_BIT;
 }
 
+bool IsFormatSupportedByPhysicalDevice(BufferType type,
+                                       VkPhysicalDevice physical_device,
+                                       VkFormat format) {
+  VkFormatProperties properties = {};
+  vkGetPhysicalDeviceFormatProperties(physical_device, format, &properties);
+
+  VkFormatFeatureFlagBits flags = VK_FORMAT_FEATURE_VERTEX_BUFFER_BIT;
```

Comment:
This should be `flag` as there is only ever one value set.

Reply:
done